apps/src/aiTutor/helpers/aiTutorContextHelper.ts

```diff
@@ -1,18 +1,45 @@
 import {AiTutorContext, MaybePromise} from '../types';
 
+const USER_SELECTION_INTRO =
```

Comment:
I think this refactor will conflict with [this pr](https://github.com/code-dot-org/code-dot-org/pull/69002), but it should be easy enough to resolve (I can do it on my end if you are ready to merge)